geometry_msgs/Pose tracker_1_pose
geometry_msgs/Pose tracker_2_pose
